src/nv_ingest/modules/filters/image_filter.py

```diff
@@ -153,7 +153,7 @@ def _apply_filter(ctrl_msg: ControlMessage, task_params: dict):
             mdf.loc[
                 filtered_images_gdf["document_type"].index, "document_type"
             ] = ContentTypeEnum.INFO_MSG.value  # noqa
-            mdf["metadata"] = mdf[exploded_metadata_cols + ["info_message_metadata"]].to_struct()  # noqa
+            mdf["metadata"] = mdf[exploded_metadata_cols].to_struct()  # noqa
```

Comment:
I'll be honest I don't fully understand why this was altered but fully trust you do. I was never super up to date on the gpu dataframe structure anyway.

Reply:
exploded_metadata_cols already has "info_message_metadata" in it, and was causing the unit tests to fail. This is a good point though.

@mpenn Could you comment here on the genesis of this? Would you prefer I did something like set(list(...)) vs assuming "info_message_metadata" is in exploded_metadata_cols